src/components/Review/ReviewSectionRowAction.tsx

```diff
@@ -192,11 +192,11 @@ const SelfAssignButton: React.FC<ReviewSectionComponentProps> = ({
   const [assignmentError, setAssignmentError] = useState(false)
 
   // Do auto-assign for other sections when one is selected
-  // for auto-assignment in another row when shouldAssign == true
+  // for auto-assignment in another row when shouldAssign == 1
   // Note: This is required to be passed on as props to be processed
   // in each row since the fullStructure is related to each section
   useEffect(() => {
-    if (shouldAssign == true) {
+    if (shouldAssign == 1) {
```

Comment:
```suggestion
    if (shouldAssign === 1) {
```

Reply:
Also -- I know I suggested this being a number, but when is it ever greater than 1?

Reply:
No. Because this is only for Self-assignment. Which is either 1 or 0 (before it's self-assigned)

Reply:
Okay, so this value can be more than 1 for other types of assignments?

Reply:
Yes, when it's received in a different place it would be including the selected id for assignee

Reply:
Okay, thanks.